<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SettingLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FRIZON</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="FontGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fonts</string>
    </property>
    <widget class="QGroupBox" name="ButtonFontGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Button Font</string>
     </property>
     <widget class="QFontComboBox" name="ButtonFontComboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="ButtonSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="TopLabelGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Top Label Font</string>
     </property>
     <widget class="QFontComboBox" name="TopLabelFontComboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="TopLabelSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="OtherLabelGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Other Label Font</string>
     </property>
     <widget class="QFontComboBox" name="OtherLabelFontComboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="OtherLabelSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
